cmds/dhclient/dhclient.go

```diff
@@ -36,6 +36,7 @@ const (
 )
 
 var (
+	attempts     = flag.Int("attempts", 5, "Max number of attempts for client to receive a valid DHCP reply from server")
```

Comment:
I think this is misleading actually, and shouldn't be committed. If I did a 

dhclient -attempts=10

I would think hope it sends 10 Solicits before giving up. That's not the case here.